R/bgCol.R

```diff
@@ -62,7 +62,7 @@ bgCol.default <- function(col,...){
     y2 <- par("usr")[4]  
   }
   
-  rect(x1,y1,x2,y2,col = col,...)
+  rect(x1,y1,x2,y2,col = col, border = NA, ...)
```

Comment:
I guess I wasn't sure why we would want the line even if `bgCol` was called. We want it to do the rect fill, not draw the border too right?